modules/virt-updating-workloads.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+[id="virt-updating-workloads_{context}"]
+= Updating virtualization workloads
+
+When you upgrade {VirtProductName}, VirtualMachineInstance (VMI) workloads, such as `virt-launcher` pods, update automatically by default. You can configure workload update strategies or opt out of workload updates by editing the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+
+* To use live migration as an update method, you must first enable live migration in the cluster.
++
+[NOTE]
+====
+If a `VirtualMachineInstance` CR contains `evictionStrategy: LiveMigrate` and the VMI does not support live migration, the VMI will not update.
+====
+
+.Procedure
+
+. To open the `HyperConverged` CR in your default editor, run the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
+
+. Edit the `workloadUpdateStrategy` stanza of the `HyperConverged` CR. For example:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  workloadUpdateStrategy:
+    workloadUpdateMethods: <1>
+    - LiveMigrate <2>
+    - Evict <3>
+    batchEvictionSize: 10 <4>
```

Comment:
@davidvossel  Is it possible to set `batchEvictionSize` to something relative to the cluster size? `10` may be much on a small cluster, but too slow for a big cluster.